code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -290,14 +290,37 @@
 		return TRUE
 	return FALSE
 
+/mob/living/simple_animal/hostile/bee/consider_wakeup()
+	if (beehome && loc == beehome) // If bees are chilling in their nest, they're not actively looking for targets
+		idle = min(100, ++idle)
+		if(idle >= BEE_IDLE_ROAMING && prob(BEE_PROB_GOROAM))
+			toggle_ai(AI_ON)
+			forceMove(beehome.drop_location())
+	else
+		return ..()
```

Comment:
```suggestion
/mob/living/simple_animal/hostile/bee/consider_wakeup()
	if (!beehome || loc != beehome)
		return ..()
    // If bees are chilling in their nest, they're not actively looking for targets
    idle = min(100, ++idle)
    if(idle >= BEE_IDLE_ROAMING && prob(BEE_PROB_GOROAM))
        toggle_ai(AI_ON)
        forceMove(beehome.drop_location())
```
early returns

Reply:
grumble grumble didn't even touch the code just moved it so it made more sense (ok)